src/views/categories.py

```diff
@@ -140,6 +140,11 @@ def add_input():
     if not name:
         return False
 
+    # No slashes in category names (see: KeePassXC csv protocol)
```

Comment:
Since you truncate categories names that contain a `item['category'] = item['category'].split('/')[1]` when importing, why not doing something similar when exporting instead of preventing users to have a `/` in their category name just in case they might export to Keepass one day?

Reply:
Keepassx provides an extra degree of nestedness to the tree.  Since we have no `root/` or `gabfl/`, we can't import it.  I am not supplying it when exporting either, because KeepassXc fills it in, and doing so might complicate other clients.